trs/server/pom.xml

```diff
@@ -78,7 +78,7 @@
                   <excludes>
                     <!--jersey and jena via commons-compress-->
                     <exclude>org.osgi:org.osgi.core</exclude>
-                    <exclude>javax.servlet:javax.servlet-api</exclude>
+                    <exclude>jakarta.servlet:jakarta.servlet-api</exclude>
```

Comment:
Let's actually undo this enforcement suppression. I did it because some of our deps were using newer servlet versions and we were behind. Now that we are no longer behind, we can again enforce not declaring a servlet version older than any of our deps.

Reply:
@berezovskyi Can I leave this TRS change to you? Not sure what you are doing, so don't want to destroy it. Not sure how to test it.

Reply:
I will do the change. Maven enforce changes can be tested just using `mvn clean verify`. If the build does not fail, the enforcement exception can be removed.

Reply:
fixed